libraries/AdaptiveExpressions/BuiltinFunctions/Add.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace AdaptiveExpressions.BuiltinFunctions
+{
+    public class Add : ExpressionEvaluator
+    {
+        public Add(string alias = null)
+            : base(alias ?? ExpressionType.Add, Evaluator(), ReturnType.String | ReturnType.Number, Validator)
```

Comment:
I don't think we should add Alias as a core abstraction, i think alias should live in the "registration"  stage, ie, when you add those functions into the table, in that phase, alias is added and managed.

If we add alias as a core abstraction, what about more than one alias? you have to made it a list, which seems not elegant